src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -38,6 +38,17 @@ ascanalpha.httponlysite.otherinfo.nossl = Site has no SSL/TLS support.
 ascanalpha.httponlysite.otherinfo.redirecttohttp = Redirected to HTTP.
 ascanalpha.httponlysite.otherinfo.urinotencoded = Redirection URI couldn't be encoded.
 
+ascanalpha.httpoxy.name = Httpoxy - using the HTTP Proxy header specified in the request
+ascanalpha.httpoxy.desc = The application uses the HTTP Proxy header specified in the request. \
```

Comment:
Perhaps:
"The server initiated a proxied request via the proxy specified in the HTTP Proxy header of the request."